vault/resource_aws_auth_backend_role.go

```diff
@@ -319,16 +393,27 @@ func awsAuthBackendRoleRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("role", role)
 	d.Set("auth_type", resp.Data["auth_type"])
 
+	// Read Vault's response into the currently supported version of these fields.
+	d.Set("bound_account_ids", resp.Data["bound_account_id"])
```

Comment:
When setting aggregate types (list, set, map), we need to check the error from `d.Set` to make sure the data was correctly saved to state:

- https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#error-checking-aggregate-types